app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
```

Comment:
Avoid using class variables for storing instances of the class. This can lead to memory leaks in a real-world application. Instead, you should create a separate class or function to manage your `Person` instances. This is a code quality issue.